client/app/billing/autoRenew/disable/translations/Messages_fr_FR.json

```diff
@@ -5,6 +5,9 @@
   "billing_autorenew_disable_renew": "Il vous suffira de renouveler votre service si vous souhaitez le conserver.",
   "billing_autorenew_disable_data": "Attention, une fois le service expiré, il sera immédiatement suspendu, il ne sera plus accessible, et les données liées à ce service seront définitivement supprimées au bout de quelques jours sans règlement de votre part",
   "billing_autorenew_disable_bulk_unavailable": "Le renouvellement automatique sera désactivé pour l'ensemble de vos services sélectionnés, exceptés pour {{ count }} services :",
+  "billing_autorenew_disable_bulk_unavailable_one": "Le renouvellement automatique sera désactivé pour l'ensemble de vos services sélectionnés, exceptés pour 1 service :",
```

Comment:
```suggestion
  "billing_autorenew_disable_bulk_unavailable_one": "Le renouvellement automatique sera désactivé pour l'ensemble de vos services sélectionnés, à l'exception de ce service",
```
?

Reply:
won't match with the current display

Reply:
But isn't it better ? 😄

Reply:
Not in the current context :) we do not only display the services but the reason why they're not eligible for activation/deactivation (actually we display the reason and the customer can expand the list of impacted services). So I'm not sure this content would make sense (given that context)